package.json

```diff
@@ -42,7 +42,7 @@
   "dependencies": {
     "@amsterdam/arm-core": "^0.6.0",
     "@amsterdam/asc-assets": "^0.32.0",
-    "@amsterdam/asc-ui": "^0.32.1",
+    "@amsterdam/asc-ui": "^0.35.0",
     "@amsterdam/react-maps": "^0.12.0",
     "@datapunt/leaflet-geojson-bbox-layer": "0.1.1",
     "@datapunt/matomo-tracker-js": "^0.5.1",
```

Comment:
The number of npm vulnerabilities has increased to 18 now. I recently managed to reduce it to 8.

Reply:
When you merge develop then this will probably be resolved.

Reply:
True. I'll also create a separate PR for another vulnerability